packages/reactstrap-validation-select/README.md

```diff
@@ -52,6 +52,8 @@ Please refer to [react-select with async pagination](https://github.com/TheSharp
 - **`raw`**: Boolean. Optional. Default: `false`. If `true`, the entire object of the selected value will be returned as the value instead of the value for the `valueKey` within the object.
 - **`valueKey`**: String. Optional. The key of the value you want returned when selected. Default: `value`
 - **`labelKey`**: String. Optional. The key for the label you want to appear in the dropdown for the user to see. Default `label`
+- **`ariaFeedback`**: Object. Optional. An Object that has two properties: `feedbackId` and `errorMessage`. It maps an error message to an error id.
+- **`placeholder`**: String. Optional. A place holder for displaying error message or help message. 
```

Comment:
this is still misleading. Users should only being passing in the visible placeholder text here.

Reply:
@LauRoxx , please let me know what would you put there. thanks

Reply:
I would just say "Placeholder text for the input" without mentioning error/help messages.